### PR TITLE
FIX: Do not use operands, which is nditer-specific.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: skxray/correlation.py [hunk 1]
```diff
@@ -119,9 +119,11 @@ def multi_tau_auto_corr(num_levels, num_bufs, labels, images):
...
                          "multiple-taus (must be even)")
 
-    if labels.shape != images.operands[0].shape[1:]:
-        raise ValueError("Shape of the image stack should be equal to"
-                         " shape of the labels array")
+    if hasattr(images, 'frame_shape'):
+        # Give a user-friendly error if we can detect the shape from pims.
+        if labels.shape != images.frame_shape:
+            raise ValueError("Shape of the image stack should be equal to"
+                             " shape of the labels array")
```
Comment (tacaswell) [inline skxray/correlation.py]: <rabbit hole> this should have an `elif` looking for `shape` and then use a `tee` to check the shape of the first image </rabbit hole>

Comment (danielballan) [inline skxray/correlation.py]: Too cute for function-specific code. Maybe a good utility.

**Comment by tacaswell** [comment]
@danielballan @sameera2004 Something is very broken here.

**Comment by tacaswell** [comment]
@sameera2004 can you figure out why this broke the tests?

**Comment by sameera2004** [comment]
@tacaswell sure I will take look

**Comment by sameera2004** [comment]
@tacaswell I fixed the tests and made a PR to @danielballan branch

**Pull request merged**